Templates/SendGrid-NodeJS/function.json

```diff
@@ -10,6 +10,7 @@
             "type": "sendGrid",
             "name": "message",
             "direction": "out",
+            "apiKey": "SendGridApiKey",
             "from": "Azure Functions <samples@functions.com>",
```

Comment:
Update the metadata.json for the template to prompt for this setting as well.
https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Templates/SendGrid-NodeJS/metadata.json#L11